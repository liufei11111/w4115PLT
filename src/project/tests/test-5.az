{* hello *}
Int a;
{**}
Int main(Int argc, String argv){
{*
hello
world


*}
}