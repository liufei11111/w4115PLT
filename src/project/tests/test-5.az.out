static int a=0;

public static int main(int argc, String argv)
{
try{}catch(Exception e){return -1;}}

